build.assets/robotest_run_nightly.sh

```diff
@@ -9,21 +9,21 @@ DOCKER_STORAGE_DRIVERS="overlay2"
 declare -A UPGRADE_MAP
 
 # latest patch release on this branch, keep this up to date
-UPGRADE_MAP[6.1.18]="centos:7 ubuntu:16"
+UPGRADE_MAP[6.1.24]="centos:7 ubuntu:16"
 
 # latest patch release on previous compatible LTS, keep this up to date
-UPGRADE_MAP[5.5.37]="centos:7 ubuntu:16"
+UPGRADE_MAP[5.5.41]="centos:7 ubuntu:16"
 
 # first release from this release & last LTS, these don't need to change until a new major/minor release
 UPGRADE_MAP[6.1.0]="ubuntu:16"
 UPGRADE_MAP[5.5.0]="ubuntu:16"
 
 # important versions in the field, per:
 # https://github.com/gravitational/robotest/issues/155#issuecomment-589743687
-UPGRADE_MAP[5.5.19]="ubuntu:16"
-UPGRADE_MAP[5.5.20]="ubuntu:16"
 UPGRADE_MAP[5.5.28]="ubuntu:16"
 UPGRADE_MAP[5.5.36]="ubuntu:16"
+UPGRADE_MAP[5.5.38]="ubuntu:16"
+UPGRADE_MAP[5.5.40]="ubuntu:16"
```

Comment:
How did you determine `5.5.19 `and `.20` are no longer important? DM me if there is any customer sensitive data in this decision. I'm curious because I'd like to deprecate versions from this list in the future.

Reply:
Oh, @wadells I was just porting changes from https://github.com/gravitational/gravity/pull/1464/commits/1ee361ee5a6ab494c374bbe632cfd702381220d2

If you think these versions are important to test we can add them back in.